backend/attendance-service/src/main/java/org/egov/repository/rowmapper/AttendeeQueryBuilder.java

```diff
@@ -0,0 +1,98 @@
+package org.egov.repository.rowmapper;
+
+import org.apache.commons.lang3.StringUtils;
+import org.egov.web.models.AttendanceRegisterSearchCriteria;
+import org.springframework.stereotype.Component;
+
+import java.util.Collection;
+import java.util.List;
+
+@Component
+public class AttendeeQueryBuilder {
+
+    private static final String FETCH_ATTENDANCE_REGISTER_QUERY = "SELECT register.*," +
+            "attendee.id AS attendeeId,attendee.register_id AS attendeeRegId,attendee.individual_id AS attendeeIndividualId, attendee.enrollment_date AS attendeeEnroll, attendee.tenantid AS attendeeTenant " +
```

Comment:
Do we require register.* in this search query as we are using this query to show only attendee data

Reply:
We also need to send the register details like id, number, namer etc from the register table. Hence added register.*    
"attendanceRegister": [
    {
      "id": "64e33343-7b4c-4353-9abf-4de8f5bcd732",
      "tenantId": "pb.amritsar",
      "registerNumber": "REG/2022-23/001",
      "name": "Class-10-E Physics",
      "startDate": 1665497225000,
      "endDate": 1665497271000,
      "status": "ACTIVE",